includes/admin/reporting/export/class-batch-export-payments.php

```diff
@@ -117,10 +117,11 @@ public function get_data() {
 		if( $payments ) {
 
 			foreach ( $payments as $payment ) {
-				$payment_meta   = edd_get_payment_meta( $payment->ID );
-				$user_info      = edd_get_payment_meta_user_info( $payment->ID );
-				$downloads      = edd_get_payment_meta_cart_details( $payment->ID );
-				$total          = edd_get_payment_amount( $payment->ID );
+				$payment = new EDD_Payment( $payment->ID );
+				$payment_meta   = $payment->payment_meta;
+				$user_info      = $payment->user_info;
+				$downloads      = $payment->cart_details;
+				$total          = $payment->total;
```

Comment:
Might as well get rid of some of these variable declarations as well since they're unnecessary now.